src/rules/noVarRequiresRule.ts

```diff
@@ -28,8 +30,24 @@ export class Rule extends Lint.Rules.AbstractRule {
             In other words, the use of forms such as \`var module = require("module")\` are banned.
             Instead use ES6 style imports or \`import foo = require('foo')\` imports.`,
         optionsDescription: "Not configurable.",
-        options: null,
-        optionExamples: [true],
+        options: {
+            items: {
+                properties: {
+                    "ignore-module": {
+                        type: "array",
+                        items: {
+                            type: "string",
+                        },
+                        minLength: 1,
+                    },
+                },
+                type: "object",
+            },
+            maxLength: 1,
+            minLength: 0,
+            type: "array",
+        },
+        optionExamples: [true, [true, { [OPTION_IGNORE_MODULE]: ["\\.html$", "\\.css$"] }]],
```

Comment:
this is a bad example because html and CSS files are not appropriate for TSLint.